convergence.md

```diff
@@ -1,3 +1,445 @@
+* v0.9.9-release
+* v0.9.10-release
+* v0.9.11-release
+* v0.9.12-release
+* v0.10
+* v0.10.0-release
+* v0.10.1-release
+* v0.10.2-release
+* v0.10.3-release
+* v0.10.4-release
+* v0.10.5-release
+* v0.10.6-release
+* v0.10.7-release
+* v0.10.8-release
+* v0.10.9-release
+* v0.10.10-release
+* v0.10.11-release
+* v0.10.12-release
+* v0.10.13-release
+* v0.10.14-release
+* v0.10.15-release
+* v0.10.16-release
+* v0.10.17-release
+* v0.10.18-release
+* v0.10.19-release
+* v0.10.20-release
+* v0.10.21-release
+* v0.10.22-release
+* v0.10.23-release
+* v0.10.24-release
+* v0.10.25-release
+* v0.10.26-release
+* v0.10.27-release
+* v0.10.28-release
+* v0.10.29-release
+* v0.10.30-release
+* v0.10.31-release
+* v0.10.32-release
+* v0.10.33-release
+* v0.10.34-release
+* v0.10.35-release
+* v0.10.36-release
+* v0.10.37-release
+* v0.10.38-release
+* v0.11.0-release
+* v0.11.1-release
+* v0.11.2-release
+* v0.11.3-release
+* v0.11.4-release
+* v0.11.5-release
+* v0.11.6-release
+* v0.11.7-release
+* v0.11.8-release
+* v0.11.9-release
+* v0.11.10-release
+* v0.11.11-release
+* v0.11.12-release
+* v0.11.13-release
+* v0.11.14-release
+* v0.11.15-release
+* v0.11.16-release
+* v0.12
+* v0.12.0-release
+* v0.12.1-release
+* v0.12.2-release
+
+Tags:
+* v0.12.2
+* v0.12.1
+* v0.12.0
+* v0.11.16
+* v0.11.15
+* v0.11.14
+* v0.11.13
+* v0.11.12
+* v0.11.11
+* v0.11.10
+* v0.11.9
+* v0.11.8
+* v0.11.7
+* v0.11.6
+* v0.11.5
+* v0.11.4
+* v0.11.3
+* v0.11.2
+* v0.11.1
+* v0.11.0
+* v0.10.38
+* v0.10.37
+* v0.10.36
+* v0.10.35
+* v0.10.34
+* v0.10.33
+* v0.10.32
+* v0.10.31
+* v0.10.30
+* v0.10.29
+* v0.10.28
+* v0.10.27
+* v0.10.26
+* v0.10.25
+* v0.10.24
+* v0.10.23
+* v0.10.22
+* v0.10.21
+* v0.10.20
+* v0.10.19
+* v0.10.18
+* v0.10.17
+* v0.10.16
+* v0.10.15
+* v0.10.14
+* v0.10.13
+* v0.10.12
+* v0.10.11
+* v0.10.10
+* v0.10.9
+* v0.10.8
+* v0.10.7
+* v0.10.6
+* v0.10.5
+* v0.10.4
+* v0.10.3
+* v0.10.2
+* v0.10.1
+* v0.10.0
+* v0.9.12
+* v0.9.11
+* v0.9.10
+* v0.9.9
+* v0.9.8
+* v0.9.7
+* v0.9.6
+* v0.9.5
+* v0.9.4
+* v0.9.3
+* v0.9.2
+* v0.9.1
+* v0.9.0
+* v0.8.28
+* v0.8.27
+* v0.8.26
+* v0.8.25
+* v0.8.24
+* v0.8.23
+* v0.8.22
+* v0.8.21
+* v0.8.20
+* v0.8.19
+* v0.8.18
+* v0.8.17
+* v0.8.16
+* v0.8.15
+* v0.8.14
+* v0.8.13
+* v0.8.12
+* v0.8.11
+* v0.8.10
+* v0.8.9
+* v0.8.8
+* v0.8.7
+* v0.8.6
+* v0.8.5
+* v0.8.4
+* v0.8.3
+* v0.8.2
+
+Within joyent/node, development is currently focused on the `master`, `v0.10` and `v0.12` branches. `v0.10` is a long term support maintenance branch. `0.12` is the most recent release maintenance branch and `master` is the target for new development intended for `v0.13`
+
+Current github statistics show that the joyent/node repository has 10,604 commits, 135 total branches, 254 releases, and 597 contributors.
+
+### io.js
+
+#### Organization
+
+The github.com/iojs organization was established to support the io.js project.
+
+The organization currently contains 112 members. (https://github.com/orgs/iojs/people)
```

Comment:
Figured as much. Not being a member of the organization gives me a limited view. For instance, I cannot see any of the existing teams that may have been formed. Perhaps I could be added to the organization?